dev/devicelab/bin/tasks/flutter_test_performance.dart

```diff
@@ -54,6 +54,8 @@ Future<int> runTest({bool coverage = false}) async {
     if (step == TestStep.starting && entry == 'Building flutter tool...') {
       // ignore this line
       step = TestStep.buildingFlutterTool;
+    } else if (step == TestStep.starting && entry.contains('Shuffling test order')) {
```

Comment:
This is the additional line

Reply:
This should be reverted -- the whole point of this test is to verify that this kind of output is not added to our test output.